common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,98 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			15  = "green_tirassian"
+			15  = "cyan_silverhand"
+			20  = "purple"
+			20  = "red"
```

Comment:
Need an explanation. Why it's green, purple, and red?

Reply:
Stormwind's colours are pretty boring. Blue and yellow. No variety at all. My intent was to represent the different influences it receives over time. Purple for Kirin Tor, Red for Stromic and Green for Tirassian sailors who might have immigrated. Given our bookmark dates this can be rather anachronistic but having only two colours seems too bland as well, especially if the culture expands over a larger area.

Reply:
The culture never has just 2 colors. There are basic colors.
From what I get, immigrants should stay with their original culture and it will work as you imagine it.